### ISSUE TITLE
Do not add version number to the website title when browsing the latest version

### ISSUE THREAD

**Issue by tomasz1986** [syncthing/docs#748]
Since the recent version picker and version string additions, version numbers seem to be always added to the website title. I don't think this is really desired when linking to the generic Docs URL, i.e. without any specific version added to it.

To make it clear what I mean here, below is the title as of right now.

```
<title>Welcome to Syncthing’s documentation! &#8212; Syncthing v1.19.1-13-g0dbd5f4 documentation</title>
```

I believe this will create complexity with search engines, e.g. Google, which will probably add these strings to the title there too, resulting in unnecessary noise. Also, the same thing has already been happening on the forums, where the software automatically grabs the current title and displays it as follows.

![image](https://user-images.githubusercontent.com/5626656/161115837-822f0d15-018c-4d0c-974b-3eee15c2a352.png)

This is problematic, because a) the actual URL behind the link is just [https://docs.syncthing.net/users/ignoring.html](https://docs.syncthing.net/users/ignoring.html) without any version string, and also, as far as I understand, b) the link on the forum is going to stay like that forever.

In sum, I'd propose to add the version string to the title only when we're actually browsing an older version of the Docs, and skip it when browsing the generic Docs URL or the `latest` version.

**Comment by acolomb** [comment]
That was an open question I raised in the original PR: https://github.com/syncthing/docs/pull/729#issuecomment-1038443251

It's basically the way Sphinx treats the corresponding variables. I don't know if the template for the page title can be overridden to use the other variable or none at all. Probably so, and that would be a good solution I guess. Switching the content of the variables contrary to their documented purpose may have strange effects when the theme gets updated sometime.

Would you rather have a shorter version string or no version at all in the title?

**Comment by calmh** [comment]
These can be different in different builds. We used to just set it to `v1`. We can still do that by default, and only populate the `version` and `release` variables when `git describe` == `git describe --abbrev=0` (that is, when we are building something that is precisely on a tag)

**Comment by acolomb** [comment]
> The "latest" version will generally not be.

Not quite true in the period after tagging a docs release, but before the next docs commit is made.  But I like the idea, so implemented a slight variation of it in #749.

**Comment by calmh** [comment]
That's what "generally" means. :)
